### ISSUE TITLE
Don't use attributes without names in abstract objects

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/jeo-maven-plugin#640]
We have several places where we use attributes without names in abstract objects. For example, we do it with methods:
```
new-KClW ↦ ⟦
  Φ.org.eolang.int(
    α0 ↦ Φ.org.eolang.bytes(
      Δ ⤍ 00-00-00-00-00-00-00-00
    )
  )
⟧
```

`Φ.org.eolang.int` is an attribute without name. This is [forbidden](https://github.com/objectionary/eo/issues/3294). We need to find all the places where we do it and fix them.

This issue blocks [normalizer](https://github.com/objectionary/normalizer/issues/379#issuecomment-2245619152).

Similar to https://github.com/objectionary/opeo-maven-plugin/issues/350

**Comment by 0pdd** [comment]
@volodya-lombrozo 2 puzzles [#646](https://github.com/objectionary/jeo-maven-plugin/issues/646), [#647](https://github.com/objectionary/jeo-maven-plugin/issues/647) are still not solved.

**Comment by volodya-lombrozo** [comment]
@rultor release, tag is `0.5.1`, title is `0.5.1`.

**Comment by 0pdd** [comment]
@volodya-lombrozo all 2 puzzles are solved here: [#646](https://github.com/objectionary/jeo-maven-plugin/issues/646), [#647](https://github.com/objectionary/jeo-maven-plugin/issues/647).
